src/pages/Home.tsx

```diff
@@ -127,6 +134,18 @@ const Home = () => {
 
           <IosBottomSheet title={"Add Torrent"} disclosure={addModalDisclosure}>
             <VStack gap={4}>
+              <FormControl isDisabled={!Categories.length}>
```

Comment:
nitpick, but thoughts on not rendering anything if they don't have any categories setup? I'm merging anyway, just wanted your opinion on that difference

Reply:
I don't think there's much difference. But from my experience with applications, I usually expect the field to be present even if it is inactive.The lack of a field may be confusing to some

In our case it can be hidden for the sake of design purity since not everyone uses categories 🙂